docs/bis/ec/ec-views.md

```diff
@@ -0,0 +1,78 @@
+# ECViews
+
+A view is a special kind of [ECEntityClass](ec-entity-class.md). The concept is realized through an [ECCustomAttribute](ec-custom-attributes.md) which is defined in the `ECDbMap` schema.
+
+The custom attribute allows defining an abstract entity class which is not backed by actual data but rather by an ECSQL query which is executed to obtain instances of the class.
```

Comment:
This section should start with something that better describes what ECViews are.  For example if we take part o the third sentence that might make a good start.  'Allows defining an abstract entity class which is not backed by actual data but rather by an ECSql query which is executed to obtain instances of the class.'  Maybe start with the first sentence put some of the third second and then explain that it is implemented via a mapping custom attribute.

Reply:
Rewrote the section, check ECSqlReference/View.md